### PR TITLE
treehouses wifistatus (fixes #72)

### PR DESCRIPTION
**Pull request by CalebProvost**
```
Usage: treehouses wifistatus

Displays signal strength in dBm and layman nomenclature

  Example:
  treehouses wifistatus
    Error: no wifi device is present

  treehouses wifistatus
    Signal strength is -40dBm
    You have a perfect signal
```

### CODE REVIEW

**Comment by xavierh93** [comment]
My wifi signal came out to -28dBm so I didn't get a message in layman terms since I fell out of the range in your code, not sure if this matters to you. Otherwise it works

**Comment by irisb1701** [comment]
```
root@treehouses:/home/pi/cli# ./cli.sh wifistatus
Signal strength is -47 dBm  
You have an excellent signal
```
works fine
